scripts/tests/test262/run_babylon_test262_utils.js

```diff
@@ -7,10 +7,10 @@ const pfs = {
   readFile: promisify(fs.readFile),
   writeFile: promisify(fs.writeFile),
   readdir: promisify(fs.readdir),
-  stat: promisify(fs.stat)
+  stat: promisify(fs.stat),
 };
 
-const parse = require("..").parse;
+const parse = require("babylon").parse;
```

Comment:
Will this resolve parse from packages/babylon?

Reply:
I thought lerna or yarn will symlink, but they don't currently, fixed.